lms/static/scripts/frontend_apps/components/ErrorDisplay.js

```diff
@@ -41,11 +41,14 @@ ${details}
         {message}: <i>{error.message}</i>
       </p>
       <p>
-        If the problem persists{' '}
-        <a href={supportLink} target="_blank" rel="noopener noreferrer">
-          send us an email
-        </a>{' '}
-        or <a href="https://web.hypothes.is/get-help/">open a support ticket</a>
+        If the problem persists <a href={supportLink}>send us an email</a> or{' '}
```

Comment:
In other apps I think we have generally not used `target="_blank"` for email links, but I think there was an issue if that link happens to be handled by a browser-based email client (eg. if you configure Chrome to open mailto: links with Gmail for example).

Reply:
Yep, changed it to open all email links in new tabs instead, which is needed for webmail clients